lib/hammer_cli_foreman_rh_cloud.rb

```diff
@@ -5,6 +5,8 @@ module HammerCLIForemanRhCloud
   require 'hammer_cli_foreman_rh_cloud/version'
   require 'hammer_cli_foreman_rh_cloud/i18n'
   require 'hammer_cli_foreman_rh_cloud/insights'
+  require 'hammer_cli_foreman_rh_cloud/command_extensions'
+  require 'hammer_cli_foreman_rh_cloud/ping'
```

Comment:
I suppose the `command_extensions/ping` file should require the implementation class.

Reply:
No, it is required here.